packages/cli/resources/change-report/findings/details.hbs

```diff
@@ -51,30 +51,26 @@
 
 </details>
 {{/inline}}
-{{#with findingDiff}}
-  {{#if (length newFindings resolvedFindings)}}
-<h2 id="findings">Findings</h2>
-
-    {{#with newFindings}}
-### :beetle: New findings ({{ length this}})
-
-      {{#each this}}
+{{#if (length findingDiff.newFindings findingDiff.resolvedFindings)}}
+<h2 id="{{ metadata.anchor }}">{{ metadata.title }}</h2>
+  {{#if (length findingDiff.newFindings) }}
+### {{metadata.emoji}} New problems detected ({{ length findingDiff.newFindings }})
+      {{#each findingDiff.newFindings}}
 {{> finding}}
       {{/each}}
 
-    {{/with}}
+  {{/if}}
 
-    {{#with resolvedFindings}}
-### :tada: Resolved findings ({{ length this}})
+  {{#if (length findingDiff.resolvedFindings) }}
+### :tada: Problems resolved ({{ length findingDiff.resolvedFindings}})
 
-      {{#each this}}
+      {{#each findingDiff.resolvedFindings}}
 {{> finding}}
       {{/each}}
 
-    {{/with}}
+  {{/if}}
 
-    {{#if pruned}}
-Because there are so many new and resolved findings, some of them are not listed in this report.
-    {{/if}}
+  {{#if findingDiff.pruned}}
+Because there are so many new and resolved {{ metadata.name }}, some of them are not listed in this report.
```

Comment:
I know this phrasing didn't change in this PR, but it could benefit from being phrased a bit softer, maybe:
```Because of the number of {{ metadata.name }}, some of them are not listed in this report.```

Reply:
Actually scratch that, my proposal isn't helpful.